annotations/views/project_views.py

```diff
@@ -179,3 +185,67 @@ def list_projects(request):
         'projects': qs,
     }
     return render(request, template, context)
+
+
+
+def upload(request):
```

Comment:
only owner should be allowed to do this.

Reply:
also, bad view name. this is for importing appellations, not for uploading (although it does contain an upload, this is not it's main purpose).

Reply:
changed and fixed

Reply:
Changed both